docassemble/AssemblyLine/data/questions/al_visual.yml

```diff
@@ -73,15 +73,29 @@ code: |
   if _internal.get('steps') > 1:
     menu_items = al_menu_items
   else:
-    menu_items = al_menu_items[1:] # Skip the "start over" menu item
+    # Skip the "start over", download, and exit and erase menu item
+    if get_config("assembly line", {}).get("enable incomplete downloads"):
+      menu_items = al_menu_items[3:]
+    else:
+      menu_items = al_menu_items[2:]
```

Comment:
Do we want this in the config, or do we want it per-interview? I can easily imagine someone wanting it to apply to some interviews but not others.

Reply:
That's fair, I can make it two level